user_docs/en/userGuide.md

```diff
@@ -44,13 +44,13 @@ Major highlights include:
 
 * Operating Systems: all 32-bit and 64-bit editions of Windows 10, Windows 11, and all Server Operating Systems starting from Windows Server 2016.
   * both AMD64 and ARM64 variants of Windows are supported, including Copilot+ PCs.
-  * Note that 32-bit operating systems are no longer under active support.
-  * Note that Windows 10 versions earlier than 21H2 and Windows Server versions older than 2022 are no longer under active support.
+  * Note that 32-bit operating systems are no longer under active support (Windows 10 is the last 32-bit operating system).
```

Comment:
I don't think we should bother updating this line as we will remove it soon, and it's not necessary to clarify in the first place.

```suggestion
  * Note that 32-bit operating systems are no longer under active support.
```